test/test_autograd.py

```diff
@@ -7052,6 +7052,33 @@ def fn():
         # context manager. If you do, there is something wrong with the
         # locking of the forward ad level most likely
 
+    def test_maximum_minimum(self):
```

Comment:
We've been moving op-specific gradient tests out of test_autograd, maybe we could put this one in test_binary_ufuncs?

Reply:
I moved the test to test_binary_ufuncs, thanks for the suggestion!